api/include/opentelemetry/trace/span_context.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+//#include <cstdint>
+//#include <cstring>
+
+#include "opentelemetry/nostd/unique_ptr.h"
+#include "opentelemetry/trace/span_id.h"
+#include "opentelemetry/trace/trace_flags.h"
+#include "opentelemetry/trace/trace_id.h"
+#include "opentelemetry/trace/trace_state.h"
+
+namespace opentelemetry
+{
+namespace trace
+{
+
+// SpanContext contains the state that must propagate to child Spans and across
+// process boundaries. It contains the identifiers TraceId and SpanId,
+// TraceFlags, TraceState, and whether it has a remote parent.
+class SpanContext final
+{
+public:
+  // An invalid SpanContext.
+  SpanContext() noexcept : trace_state_(new TraceState) {}
+
+  // TODO
+  //
+  // static SpanContext Create(TraceId traceId, SpanId spanId, TraceFlags traceFlags, TraceState
```

Comment:
Are these definitions going to be up eventually? I need these for my HttpTraceConotext implementation. Thanks